drum/links/tests/tests_forms.py

```diff
@@ -0,0 +1,38 @@
+from unittest import TestCase
```

Comment:
No need for a whole `tests` package, let's just have a single `tests.py` module with both test classes in it.